code/obj/artifacts/artifact_items/handheld_storage.dm

```diff
@@ -0,0 +1,180 @@
+// bag of holding artifact
+// handheld storage with various types. some can be held on back or belt, in which case they "transform" to fit there, with an icon change
+// putting a bags of holding into another is bad
+
+/obj/item/artifact/bag_of_holding
+	name = "artifact bag of holding"
+	associated_datum = /datum/artifact/bag_of_holding
+	w_class = W_CLASS_NORMAL
+	/// original icon state, without artifact activation fx
+	var/base_icon_state
+	/// original item_state
+	var/base_item_state
+	/// worn belt icon states
+	// TODO: sprite eldrich and wizard belts
```

Comment:
?

Reply:
Meant it as a todo for the future, if anyone can sprite it (didn't think I could very well), I'll reclarify